src/vuex/modules/intercom.js

```diff
@@ -0,0 +1,33 @@
+import config from "src/../config"
+import { Intercom } from "capacitor-intercom"
+
+let intercom = null
+if (config.mobileApp) {
+  intercom = new Intercom()
+  const userId = "lunie-app-" + Math.floor(Math.random() * 10000 + 1).toString()
+  intercom.registerIdentifiedUser({ userId })
+}
+
+export default () => {
+  return {
+    state: {
+      intercom
+    },
+    mutations: {
+      displayMessenger(state) {
+        state.intercom.displayMessenger()
+      },
+      displayHelpCenter(state) {
+        state.intercom.displayHelpCenter()
```

Comment:
what does this do?

Reply:
It opens the help center (I guess is the same thing you talked about the last demo day conf). 

What do you think about? We can remove it if it's not going to be useful.

Reply:
as long as it is not used, we should not have it in code. But nice to know this is possible.